geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHashesIntegrationTest.java

```diff
@@ -373,23 +373,32 @@ public void testHVals() {
     String key = "HVals_key";
     String field1 = "field_1";
     String field2 = "field_2";
-    String value = "value";
+    String value1 = "value_1";
+    String value2 = "value_2";
 
-    List<String> list = jedis.hvals(key);
-    assertThat(list == null || list.isEmpty()).isTrue();
+    List<String> list = jedis.hvals("non-existent-key");
+    assertThat(list == null);
```

Comment:
Is there a reason for doing  `assertThat(list == null);` vs `assertThat(list).isEmpty();` here?

Reply:
Ah - good catch. It didn't actually even occur to me that the `assertThat`, as it is here, does nothing :(. Now fixed.